### PR TITLE
RSVP Feature

### PR DESCRIPTION
**Pull request by StevenGrimshaw**
- quick fixes for rsvp unit tests
- converted sign in naming to rsvp
- rsvp functionality
- **can only create rsvp by clicking on button by event**

### CODE REVIEW

File: app/views/dashboards/show.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <%= link_to 'Event Types', event_types_path %><br>
 <%= link_to 'Events', events_path %><br>
 <%= link_to 'Roles', roles_path %><br>
-<%= link_to 'Sign Ins', sign_ins_path %><br>
+<%= link_to 'Sign Ins', rsvps_path %><br>
```
Comment (ThomasGGoodwin) [inline app/views/dashboards/show.html.erb]: Rename the link text to RSVPs.

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@ en:
       locked: "Your account is locked."
       last_attempt: "You have one more attempt before your account is locked."
       not_found_in_database: "Invalid %{authentication_keys} or password."
-      timeout: "Your session expired. Please sign in again to continue."
-      unauthenticated: "You need to sign in or sign up before continuing."
+      timeout: "Your session expired. Please rsvp again to continue."
```
Comment (ThomasGGoodwin) [inline config/locales/devise.en.yml]: Switch these back to "sign_in".

File: config/locales/devise.en.yml [hunk 3]
```diff
@@ -44,18 +44,18 @@ en:
       signed_up_but_unconfirmed: "A message with a confirmation link has been sent to your email address. Please follow the link to activate your account."
       update_needs_confirmation: "You updated your account successfully, but we need to verify your new email address. Please check your email and follow the confirmation link to confirm your new email address."
       updated: "Your account has been updated successfully."
-      updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to sign in again."
+      updated_but_not_signed_in: "Your account has been updated successfully, but since your password was changed, you need to rsvp again."
```
Comment (ThomasGGoodwin) [inline config/locales/devise.en.yml]: ^^

File: config/locales/devise.en.yml [hunk 4]
```diff
@@ -44,18 +44,18 @@ en:
...
       already_signed_out: "Signed out successfully."
     unlocks:
       send_instructions: "You will receive an email with instructions for how to unlock your account in a few minutes."
       send_paranoid_instructions: "If your account exists, you will receive an email with instructions for how to unlock it in a few minutes."
-      unlocked: "Your account has been unlocked successfully. Please sign in to continue."
+      unlocked: "Your account has been unlocked successfully. Please rsvp to continue."
   errors:
     messages:
-      already_confirmed: "was already confirmed, please try signing in"
+      already_confirmed: "was already confirmed, please try rsvpg in"
```
Comment (ThomasGGoodwin) [inline config/locales/devise.en.yml]: ^^

**Review by ThomasGGoodwin — CHANGES_REQUESTED** [review]
See comments in code.

**Review by ThomasGGoodwin — APPROVED** [review]

**Review by ThomasGGoodwin — APPROVED** [review]

**Review by LukeY23 — APPROVED** [review]
it looks good to me

**Pull request merged**